sql/core/src/test/scala/org/apache/spark/sql/sources/v2/DataSourceV2SQLSuite.scala

```diff
@@ -162,4 +162,18 @@ class DataSourceV2SQLSuite extends QueryTest with SharedSQLContext with BeforeAn
       conf.setConfString("spark.sql.default.catalog", originalDefaultCatalog)
     }
   }
+
+  test("drop table from catalog") {
```

Comment:
Nit: naming convention is `LogicalPlanName: case description`, like:
* `"DropTable: basic behavior"`
* `"DropTable: if exists"`
* `"DropTable: when table is missing"`

I think that these should be separate cases, instead of including the second `DROP TABLE` command below.

Reply:
renamed.